rabbitmq.drush.inc

```diff
@@ -80,7 +79,7 @@ function drush_rabbitmq_queue_info($queueName = NULL) {
   }
 
   /* @var \Drupal\Core\Queue\QueueFactory $queueFactory */
-  $queueFactory = (new Drupal())->service('queue');
+  $queueFactory = \Drupal::service('queue');
```

Comment:
drupalcs throws warnings on static method calls, hence the silly looking code creating a Drupal instance.